numpy/core/_add_newdocs.py

```diff
@@ -2818,18 +2818,17 @@ def luf(lamdaexpr, *args, **kwargs):
     --------
     >>> A = np.array([1, 256, 8755], dtype=np.int16)
     >>> map(hex, A)
-    ['0x1', '0x100', '0x2233']
+    <map object at 0x...>
```

Comment:
I don't think this was the intent of the test - add a `list` on the line above instead. Same below